### PR TITLE
Update docs for v1.5 library; doc bug fixes

### PR DESCRIPTION
**Pull request by d45**
This PR is a doc update:
* What's New topic (for v1.5)
* Pause before loading debugging change (2021.1)
* Broken links and bug fixes
* OAuth sequence diagram

### CODE REVIEW

File: docs/trex_release-notes.md [hunk 1]
```diff
@@ -11,9 +11,44 @@ layout: docs
...
+
+* The following are all improvements to the [`getSummaryDataAsync`](https://tableau.github.io/extensions-api/docs/interfaces/worksheet.html#getsummarydataasync){:target="_blank"} method:
+
+  * `getSummaryDataAsync` now has a smaller and faster payload.
+
+  * `maxRows` can be applied to `getSummaryDataAsync` to restrict the number of rows fetched.
+
+  * `columnsToIncludeById` can be applied to `getSummaryDataAsync` to restrict the columns fetched.
+
+  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryData` to restrict the amount of information returned to what you really need.
```
Comment (seanmakesgames) [inline docs/trex_release-notes.md:38]: add Async to end of this reference
`getSummaryDataAsync`

Comment (d45) [reply docs/trex_release-notes.md]: Argh! John caught these errors earlier, but I missed this one reference (fixed it now, asynchronously).  Thanks!

**Review by seanmakesgames — DISMISSED** [review]

File: docs/trex_release-notes.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ About this release:
 
   * `columnsToIncludeById` can be applied to `getSummaryDataAsync` to restrict the columns fetched.
 
-  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryData` to restrict the amount of information returned to what you really need.
+  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryDataAysnc` to restrict the amount of information returned to what you really need.
```
Comment (johnDance) [inline docs/trex_release-notes.md:38]: Sorry... one more. It should by Async, rather than Aysnc.

Comment (d45) [reply docs/trex_release-notes.md:38]: Right...!  fixing it now. Thanks.

Comment (d45) [reply docs/trex_release-notes.md]: Fixed. Thanks for the corrections!

**Review by johnDance — APPROVED** [review]

**Pull request merged**